modules/cnv-dedicated-resources.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-edit-vms.adoc
+// * cnv/cnv_vm_templates/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-dedicated-resources_{context}"]
+
+= Enabling dedicated resources for a {object}
+
+Use this procedure to enable dedicated resources for a {object}.
+
+When you enable dedicated resources for your {object},
+you obtain CPU time from a specific CPU or set of CPUs, maximizing efficiency
+while optimizing usage of system resources.
+
+.Procedure
+
+To enable dedicated resources, do the following:
+
+. Ensure that the `cpumanager` node label for the CPU resource you want to
+dedicate is set to `true`. If you do not do this,
+you will receive an error message when you attempt to dedicate the resource,
+indicating that no qualified nodes with the required labels (`cpumanager=true`)
+exist.
```

Comment:
It is not an error, it is just a warning. You can still save the change, just when you attempt to start the VM, it will not get scheduled.

Reply:
yes we should add that the rule can still be applied it will just impact scheduling of the workload.

Reply:
Let me know what you think of this and if he need to be more clear. Right now I think this is verging on wordy and may attempt to cut it down some more --- granted it is complex, but...: 

". Ensure that the `cpumanager` label is set to `true` for each node on which you want
to schedule virtual machines with dedicated resources. If you do not set the label appropriately,
you will receive an warning message when you attempt to dedicate the resource,
indicating that no qualified nodes with the required labels (`cpumanager=true`)
exist. As a result the workload will not be scheduled."

Reply:
> If you do not set the label appropriately

this is not entirely correct, you do not set the label. You configure the cpumanager on the node and the fact you have it correctly configured causes the label to appear. I would put it:
....
If the label is not present (e.g. the cpumanager is not properly configured on the particular node), you will receive a warning message when you attempt to dedicate the resource,
...

Reply:
Got it, thanks. I won't refer to labels being set.